app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -180,5 +177,19 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$scope', 'miqServ
     }
   };
 
+  var formOptions = function() {
+    API.get("/api/providers?expand=resources&attributes=id,name&filter[]=supports_block_storage?=true").then(function(data) {
```

Comment:
Yes. This code is currently only replacing the existing logic which filtered with `supports_block_storage?`. In EBS, `supports_create?` is only available in the `CloudVolume` so I am not sure how to check for that or even if it makes sense at this point.